content/substrate/code/slides.md

```diff
@@ -152,7 +152,7 @@ Bute then each node can decide which subset of these it chooses, and how it impl
 
 Notes:
 
-<https://crates.io/crates/substrate-wasm-builder> (seen env variables, pretty useful!) <!-- markdown-link-check-disable-line -->
+<https://crates.io/crates/substrate-wasm-builder> (seen env variables, pretty useful!)
```

Comment:
We cannot disable a check per line now, only a glop pattern for mlc config... [for now](https://github.com/becheran/mlc/issues/76)